UI/src/utils/styleRule.js

```diff
@@ -110,6 +110,16 @@
             return this;
         };
 
+      this.isBetween = function(values){
+        var expression = expressionFn[expressionFn.length-1];
+        if(!expression || expression.compare)
+          throw 'You must have on of the following functions ["where", "and", "or"] before use the "isBetween".';
+        expression.compare = function(propertyValue){
+            return values[0] <= propertyValue  && propertyValue < values[1];
```

Comment:
Should check the length of the values and throw a exception if it's diferente then 2.